### ISSUE TITLE
Unable to compile on Windows

### ISSUE THREAD

**Issue by shiioo** [bytemine/bytemine-manager#3]
Hi there!

I'm trying to compile under Windows, but I'm coming across issues that prevent ant from compiling.
I tried using JDK7 and JDK8. I'm also using the last ant version (v1.9.7).

I got this warning : [[javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6](url)
And I get an error related to ant editing the "Configuration.java" file and messing the output : [[javac] D:\bytemine\src\net\bytemine\manager\Configuration.java:90: error: unclosed string literal
[javac] public String MANAGER_BUILD = "2f00a3d1c008e04b2a0a1d25c8e10763ed0f945e
[javac]                               ^
[javac] D:\bytemine\src\net\bytemine\manager\Configuration.java:91: error: unclosed string literal
[javac] ";](url)

It edits the file and put "; on another line which triggers the error.

**Comment by fkr** [comment]
Hi,

it should build now on windows. Thanks for the report!
Problem was that we did not strip the newline properly from the git-revision file.

Once that was adjusted the build had a problem with the docs as well (since we call 'make' on a makefile to compile the tex files), I disabled this on windows.

You find the documentation here: https://www.bytemine.net/files/bytemine-manager-manual-en.pdf

regards,

felix
